src/util/__test__/sortCollectionSet.test.ts

```diff
@@ -1,6 +1,6 @@
 import { expect, it, describe } from 'vitest';
 
-import sortCollectionSet from '../sortCollectionSet';
```

Comment:
Not a fan of completely renaming a function to something with a different meaning